scripts/oe-env-prepare.ps1

```diff
@@ -0,0 +1,446 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Naming convention: all scripts should use the _verb-noun_ pattern. For example: `install-windows-prereqs.ps1`